### ISSUE TITLE
appscale up problems

### ISSUE THREAD

**Issue by khaledkbadr** [AppScale/appscale#2290]
Hi, I'm trying to get appscale running into docker. 
I installed appscale successfully, but when I executed `appscale up` I found those problems

* No file in path `/etc/appscale/VERSION/ > /root/appscale/.appscale/VERSION`
```
*** ShellException: Executing command 'ssh -F /dev/null -i /root/.appscale/appscale514146ba0db74a59a59acf5e79601681.key -o LogLevel=quiet -o NumberOfPasswordPrompts=0 -o StrictHostkeyChecking=no -o UserKnownHostsFile=/dev/null root@172.17.0.2 bash cat /etc/appscale/VERSION' failed:
cat: /etc/appscale/VERSION: No such file or directory
```

* No directory in path `/etc/appscale/certs/ > /root/appscale/.appscale/certs`
```
Executing command 'scp -r -i /root/.appscale/appscale514146ba0db74a59a59acf5e79601681.key -o LogLevel=quiet -o NumberOfPasswordPrompts=0 -o StrictHostkeyChecking=no -o UserKnownHostsFile=/dev/null /root/.appscale/appscale514146ba0db74a59a59acf5e79601681-cert.pem root@172.17.0.2:/etc/appscale/certs/mycert.pem' failed:
scp: /etc/appscale/certs/mycert.pem: No such file or directory
```

* `monit` was not configured correctly
```
ssh -F /dev/null -i /root/.appscale/appscalea35ff84f8a9e461ba2febd3276447350.key -o LogLevel=quiet -o NumberOfPasswordPrompts=0 -o StrictHostkeyChecking=no -o UserKnownHostsFile=/dev/null root@172.17.0.2 bash monit start -g controller' failed:
monit: Cannot connect to the monit daemon. Did you start it with http support?
```

* It hangs here, I tried to check the logs in `/var/logs/appscale` but found it empty. Looks like it hangs on the AppController
```
Starting AppController at 172.17.0.2
Please wait for the AppController to finish pre-processing tasks.
```
monit logs was showing: `error : 'controller-17443' failed to start `

and after a while it showed this error
```
Port 172.17.0.2:17443 did not open in time. Aborting...
```
I used **Docker version 1.13.0** on **macOS Sierra**
Thanks

**Comment by obino** [comment]
Hello Khaled,

yeah there won't be logs, since the process didn't go as far as trying
to start the AppController (the first logs will be the controller.log).

How did you compiled appscale and in which environment? It seems that
something probably went wrong during compilation, and not all the
proper files have been put in place.

cheers
graziano

**Comment by khaledkbadr** [comment]
Hey Graziano,
I installed by executing appscale_build.sh on both appscale and appscale-tools of the master branches

**Comment by obino** [comment]
Hi @khaledkbadr, did you manage to solve the problem? What was the issue?

**Comment by khaledkbadr** [comment]
Hey @obino 
what I was doing is that running  the build script from not so good Internet connection. when I ran it from good internet connection (European internet) the directories and files that were missing were present. "I didn't check monit config". But again I had another problem. I chose to use the docker container instead and get back to the installation after I checked the validity of our use case for Appscale.
